### PR TITLE
[FRONT-344] Remove canvases and dashboards

### PR DESCRIPTION
**Pull request by juhah**
Remove canvases & dashboards. Summary:

- Removed Canvas & Dashboard userpages sections
- Remove Ethereum Canvas keys from profile
- Remove unnecessary dependencies that were needed for the canvas and dashboard editor
- Restore Husky dependency that was previously removed (used for linting Git hook)
- Remove module docs and mentions of Canvases & Dashboards from docs (though some images need to be updated to completely remove them)

Sorry for the sizeable PR but it's mostly removals, have a look and see if things still work 🕵️

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -55,7 +55,8 @@ jobs:
     - WEB3_REQUIRED_NETWORK_ID=1
     - WEB3_PUBLIC_HTTP_PROVIDER=https://mainnet.infura.io/v3/17c3985baecb4c4d94a1edc2c4d23206
     - WEB3_PUBLIC_WS_PROVIDER=wss://mainnet.infura.io/ws/v3/17c3985baecb4c4d94a1edc2c4d23206
-    - SENTRY_DSN=https://e012761929d24baa96ca82772f60b395@sentry.io/1203154
+    - SENTRY_DSN=https://e012761929d24baa96ca82772f60b395@o151964.ingest.sentry.io/1203154
+    - SENTRY_INDEXER_DSN=https://8311f8e7df9046b781600f95eefd1aa0@o151964.ingest.sentry.io/5235991
```
Comment (juhah) [inline .travis.yml:59]: @fonty1 Ended up with a separate variable, it won't be enabled by defualt locally but defined in `travis.yml` so should work in prod and push to the indexer Sentry project

File: app/package.json [hunk 2]
```diff
@@ -170,18 +168,13 @@
     "flow-babel-webpack-plugin": "^1.1.1",
     "flow-bin": "^0.113.0",
     "git-revision-webpack-plugin": "^3.0.6",
-    "highcharts": "^8.2.2",
-    "highcharts-react-official": "^3.0.0",
     "history": "^4.10.1",
-    "hoist-non-react-statics": "^3.3.2",
     "html-webpack-plugin": "^3.2.0",
-    "html2plaintext": "^2.1.2",
+    "husky": "^1.3.1",
```
Comment (juhah) [inline app/package.json:173]: `husky` and `lint-staged` were used for the git commit hook so brought them back

File: app/src/app/Marketplace.jsx [hunk 3]
```diff
@@ -16,6 +16,7 @@ import routes from '$routes'
 const Route = withErrorBoundary(ErrorPage)(RouterRoute)
 
 const MarketplaceRouter = () => ([
+    <Route exact path={routes.root()} component={Products} key="root" />, // edge case for localhost
```
Comment (juhah) [inline app/src/app/Marketplace.jsx:19]: This was needed when logging out and returning to product listing, without this it displays an error (before part of the editor router for some reason)

**Review by mondoreale — APPROVED** [review]
This is huge, also literally! Compiles and works as if nothing was there. Well played.

**Pull request merged**